src/state/lists/wrappedTokenInfo.ts

```diff
@@ -26,7 +29,8 @@ export class WrappedTokenInfo extends Token {
       tokenInfo.name,
     )
     this.tokenInfo = tokenInfo
-    this.list = list
+    const { name, timestamp, version, keywords, tags, logoURI } = list
+    this.list = { name, timestamp, version, keywords, tags, logoURI }
```

Comment:
const {tokens, ...restOfList} = list
this. list = restOfList

Reply:
I changed the constructor to use `LiteTokenList` as param, so `tokens` is not available now.